feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.spec.ts

```diff
@@ -379,7 +448,62 @@ describe('ConfigProductTitleComponent', () => {
       );
     });
 
-    it('should not emit in case an order bound configuration does not have the OV aspect (yet)', () => {
+    it('should get product name as part of product configuration, in case configuration is saved cart bound via config product code', () => {
```

Comment:
```suggestion
    it('should get product name via config product code in case configuration is saved cart bound and product code is not provided with routing data', () => {
```